modules/kibana/build.gradle

```diff
@@ -23,7 +23,7 @@ esplugin {
 }
 
 dependencies {
-  compile project(path: ':modules:reindex', configuration: 'runtime')
+  api project(path: ':modules:reindex', configuration: 'runtimeElements')
```

Comment:
we need to use `runtimeElements` as runtime is a) empty because it extends compile and is therefore empty now and 2) runtime is deprecated for consumption by other projects (see https://docs.gradle.org/6.5/userguide/java_library_plugin.html#sec:java_library_configurations_graph for details)

Reply:
Why are we even specifying this? Why can't we use the default configuration here?

Reply:
I don't know yet and I have that on my todo list to double check. But for the sake of keeping things simple and focussed in this PR I just did straight forward replacement for now

Reply:
I looked into this closer. I assumed there was a subtile difference between default but since we ported all dependencies from compile to api we ensured actually that they are the same here. I'll remove this